### PR TITLE
[KB] Release notes for 5/7

### PR DESCRIPTION
**Pull request by JimGreene30**
https://ctliotest.atlassian.net/browse/PE-1772

### CODE REVIEW

File: Release Notes/2019/2019-05-07-cloud-platform-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+A new feature has been on the Events Tab that includes a toggle to hide/unhide Events that have been suppressed. Green indicates Suppressed Events have been hidden.
+
+![Events1.PNG](../../images/release-notes/2019/events1.png)
+![Events2.PNG](../../images/release-notes/2019/events2.png)
+
+##### Fix to Workspace-Level Scope when adding new Policy or Events
+
+When adding a new Policy or Event, the Scope will no longer pre-populate a Workspace-Level Scope and will be required for the User to select during the creation of the Policy.
+
+![Events1.PNG](../../images/release-notes/2019/policy1.png)
```
Comment (asanokhin) [inline Release Notes/2019/2019-05-07-cloud-platform-release-notes.md]: events?

File: Release Notes/2019/2019-05-07-cloud-platform-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+![Events2.PNG](../../images/release-notes/2019/policy2.png)
```
Comment (asanokhin) [inline Release Notes/2019/2019-05-07-cloud-platform-release-notes.md]: events?

File: Release Notes/2019/2019-05-07-cloud-platform-release-notes.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+##### Access Denied Fixed for Orphaned Watcher Agents
+
+A fix was put in place to remove the error message when attempting to clean up orphaned Watcher Agents.
+
+![Events1.PNG](../../images/release-notes/2019/watcher1.png)
```
Comment (asanokhin) [inline Release Notes/2019/2019-05-07-cloud-platform-release-notes.md]: events?

File: Release Notes/2019/2019-05-07-cloud-platform-release-notes.md (hunk 1 above)
Comment (JimGreene30) [reply Release Notes/2019/2019-05-07-cloud-platform-release-notes.md]: The perils of copying code. I've updated the file names.

**Pull request merged**